README.md

```diff
@@ -35,6 +35,12 @@ Here are some examples of how you could leverage `digitalocean-cloud-controller-
 * [loadbalancers](docs/controllers/services/examples/)
 * [node labels and addresses](docs/controllers/node/examples/)
 
+## Production advise
+
+When creating load-balancers through CCM (via `LoadBalancer`-typed Services), it is important that you **must not change the DO load-balancer configuration manually.** Such changes will eventually be reverted by the reconciliation loop built into CCM. The only exception to this are load-balancer names which can be changed starting with CCM v0.1.17.
```

Comment:
Maybe link to the changelog for `CCM v0.1.17`?

Reply:
I just realized that we haven't released v0.1.17 yet, so I dropped the reference. Will re-add it along with a link to the release (that points at the change log) when I do the release PR.